extensions/ordinal.md

```diff
@@ -0,0 +1,15 @@
+# Ordinal
```

Comment:
Suggest name: Sequence

"ordinal" implies a number; however, this is a string "expressing the relative order of the event" -- this solves a broader range of use cases than just a number, and is definitely a good approach.  Propose we adjust the naming to match the definition.